ledgers/ccf/pdo/ledgers/ccf/scripts/set_contract_enclave_expected_sgx_measurements.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import http
+import sys
+
+from loguru import logger as LOG
+
+from pdo.ledgers.ccf.common import parse_common_arguments
+
+
+# -----------------------------------------------------------------
+def set_contract_enclave_expected_sgx_measurements(client, options):
+
+    params = {}
+    params['mrenclave'] = options.mrenclave
+    params['basename'] = options.basename
+    params['ias_public_key'] = options.ias_public_key
+
+    r = client.post("/app/set_contract_enclave_expected_sgx_measurements", params)
+    if r.status_code != http.HTTPStatus.OK.value:
+        LOG.error('failed to set contract enclave expected sgx measurements: {}, code: {}'.format(
+            r.body, r.status_code))
+        sys.exit(-1)
+
+# -----------------------------------------------------------------
+def Main() :
+
+    (_, unprocessed_args, member_client) = parse_common_arguments(
+        sys.argv[1:], 'Fetch the ledger authority key from a CCF server', True)
+
+    # Parse the arguments that are unique to the script
+
+    parser = argparse.ArgumentParser(description='Fetch the ledger authority key from a CCF server')
+    parser.add_argument('--mrenclave', help="Expected MRENCLAVE of pdo enclaves", type=str)
```

Comment:
this doesnt actually fetch the ledger authority key. the description needs to be fixed

Reply:
thanks, i think what is committed for the PR says "parser = argparse.ArgumentParser(description='Set contract enclave expected sgx measurements')". (there are two commits in this PR, please see the last commit). I will squash all commits before merge (given that this is a small PR, IMO we do not need more than 1 commit)